### ISSUE TITLE
Loading files format

### ISSUE THREAD

**Issue by ramzialhaddadtm** [Mapioi/OnlineLoggerPro#2]
So I am trying to use this, and test it out just out of curiosity and there seem to be no instruction on how to load data from it, I tried loading my excel sheets as the excel extension as well as the csv, the logger pro file save, and just a txt with data. Could you be more elaborate with how to use? and even provide examples?

**Comment by TrakJohnson** [comment]
Thanks for your interest ! I have indeed not documented it properly, as it is quite unstable and doesn't have many features yet, however getting started should be easy.

For now, it only supports loading the logger pro save files (`.qmbl`, `.cmbl`) and will display a table and a graph. I have just added two example files to the repo, I am sure they work. If your logger pro file save doesn't work, please post it here and I'll investigate.

**Comment by ramzialhaddadtm** [comment]
Yeah, I can't load any of my tables at all just becomes a blank screen. Attached are the tables that don't work for me. Maybe its different version of the software?

[Tables.zip](https://github.com/Mapioi/OnlineLoggerPro/files/2569339/Tables.zip)

**Comment by ramzialhaddadtm** [comment]
Your examples work great. Mine just leaves me with a blank white screen.

**Comment by TrakJohnson** [comment]
Thanks for the files, I'll have a look and tell you when its fixed. It is normal that it doesn't work with all files yet, as there is no official specification for this file format and therefore writing the parser is done through trial and error :/
